.github/workflows/build.yml

```diff
@@ -103,6 +111,12 @@ jobs:
                 testmo_token: ${{ secrets.TESTMO_TEST_TOKEN }}
                 source: 'build-test'
 
+            - name: rename for nightly
+              uses: ./.github/actions/nm-rename-for-nightly-whl/
+              if: contains(fromJSON('["NIGHTLY"]'), inputs.wf_category)
```

Comment:
The upstream `build-test` workflow also supports `WEEKLY` for `wf_category`; will that be passed through as-is? If so, `WEEKLY` should be supported here (and noted in the descriptions above).

Related, shouldn’t the `build-test` workflow be updated in this PR to pass `wf_category` when calling this workflow?

Reply:
yeah. i've update everything to be driven by ENV in `setup.py` so no need to call "rename for nightly". the default is set to always generate a "nightly". when we want to release, it needs to be explicitly specified.